.github/release/full/Dockerfile

```diff
@@ -1,17 +1,15 @@
-FROM debian:bookworm-slim
+FROM python:3.11-slim-bullseye
 
 ENV DEBIAN_FRONTEND noninteractive
 ENV TERM linux
 
+RUN apt-get update && apt-get --yes --no-install-recommends install build-essential
```

Comment:
This will defeat the single-line apt command, in order to remove the package cache without committing another storage layer. Also, the build tools will not get removed afterwards.